java/src/jmri/NamedBean.java

```diff
@@ -365,6 +365,14 @@ public void addPropertyChangeListener(@Nonnull String propertyName, @Nonnull Pro
      */
     public void setComment(@CheckForNull String comment);
 
+    /**
+     * Get a list of references to the specified bean.
+     *
+     * @param bean The bean to be checked.
+     * @return a list of NamedBeanUsageReports or an empty ArrayList.
+     */
+    default ArrayList<NamedBeanUsageReport> getUsageReport(@CheckForNull NamedBean bean) { return (new ArrayList<>()); }
```

Comment:
It's better to return a List than an ArrayList, since List is an interface and ArrayList is a concrete class.

Reply:
Changed in next commit.